src/fileformat/types/sec_seg/sec_seg.cpp

```diff
@@ -521,6 +536,27 @@ void SecSeg::setMemory(bool sMemory)
 	isInMemory = sMemory;
 }
 
+/**
+ * Compute entropy of section data in <0,1>
+ */
+void SecSeg::computeEntropy()
+{
+	if (!loaded)
+	{
+		return;
+	}
+	
+	auto data = reinterpret_cast<const uint8_t *>(bytes.data());
```

Comment:
Is `reinterpret_cast` really necessary here? Wouldn't `static_cast` be enough or is the cast necessary at all?

Reply:
Cast is necessary, since `bytes.data()` returns `const char *` and `computeDataEntropy` takes `const unsigned char*` as parameter. As stated earlier, `static_cast` doesn't suffice. `invalid static_cast from type ‘const char*’ to type ‘const uint8_t*’ {aka ‘const unsigned char*’}`, please see https://stackoverflow.com/questions/658913/c-style-cast-from-unsigned-char-to-const-char

Reply:
Also, I wouln't be so concerned about wrong interpretation of signedness as `computeDataEntropy` doesn't take it into consideration.